app/src/main/res/values/strings.xml

```diff
@@ -175,4 +175,8 @@
     <string name="Pref_General">General</string>
     <string name="Pref_Reset_Previous">Reset to previous</string>
     <string name="Pref_Reset_Default">Reset to default</string>
+    <string name="Finished_Diag_Title">Info</string>
```

Comment:
I guess that you derived the name from finishedDiag.
Can you please rename it to `Welcome_Diag_` and put it below the `<!-- Dialogs -->` section ?